packages/app/src/specs/SpecsListRunWatcher.vue

```diff
@@ -0,0 +1,50 @@
+/**
+ Non rendering component used to watch running runs and emit an event
```

Comment:
I'm fine with a non-rendering component, but is this something that might as well be a composable that exposes a ref, instead of a component that emits an event?

I have definitely made at least one renderless component mostly to follow the convention we had of doing GQL requests in SFCs, but we seem to have not maintained that convention in other place. So just curious if there's some other reason to make this a component that I'm missing? (Idk, ease of hooking into lifecycle or something?).

Reply:
I was not sure of another way of using the Urql `useSubscription` across an array of options without having this renderless component. This implementation allows it to get mounted for each run on the page after the original page has mounted and the GQL queries returned.  If anyone has another pattern for this, please share!

Reply:
This seems fine to me.